tools/packaging/kata-deploy/local-build/kata-deploy-binaries-in-docker.sh

```diff
@@ -114,6 +114,7 @@ docker run \
 	--env ARTEFACT_REGISTRY_PASSWORD="${ARTEFACT_REGISTRY_PASSWORD}" \
 	--env TARGET_BRANCH="${TARGET_BRANCH}" \
 	--env BUILDER_REGISTRY="${BUILDER_REGISTRY}" \
+	--env TEE_PLATFORM="${TEE_PLATFORM:-}" \
```

Comment:
Shouldn't we extract out `TEE_PLATFORM="${TEE_PLATFORM:-}"` like all the other envs?

Reply:
If we don't have this line, there is no way of building the guest components where `se-attester` is solely built with `make coco-guest-components-tarball`. With this line, we can achieve to have those guest components by running `make coco-guest-components-tarball TEE_PLATFORM=se`. 

If there is another way of achieving this goal, please let me know. I will following that way.

Reply:
Sorry for not being clear. The pattern in that file seems to be to set `TEE_PLATFORM="${TEE_PLATFORM:-}"` separately (e.g. https://github.com/kata-containers/kata-containers/blob/2396968d92f00636caa216675a5b2e0f85844742/tools/packaging/kata-deploy/local-build/kata-deploy-binaries-in-docker.sh#L82-L103), then in the `docker` call, just use `--env TEE_PLATFORM="${TEE_PLATFORM}"`. I don't think the effect will be different and don't necessarily even like that convention, but I think it's best to be consistent.

Reply:
yes, you are right. It does not look consistent with others. Thanks!